### ISSUE TITLE
Can't use Project ID to install mods

### ISSUE THREAD

**Issue by endrjux** [jakobkmar/pacmc#43]
It seems the new version of pacmc doesn't support installing by Project ID:

```
$ pacmc install 394468
Resolving versions and dependencies...
[✓] resolved slugs
The project 394468 could not be found
```
Adding `curseforge/` doesn't help. (394468 is a valid ID)
This worked previously.

If this feature is being dropped in new pacmc, README should be updated as it still mentions this method in the "Install a mod" section.

It would be nice if this was possible though. Example use case is migration from the old pacmc to the new one: I saved list of IDs from old mods directory and now would like to just pass those IDs to new pacmc.

**Comment by jakobkmar** [comment]
This is a bug, installing via id currently only works for Modrinth after the rewrite (0.5.0) - but I plan to reintroduce support for Curseforge IDs as well with the next release.

I am sorry for the inconvenience, if you have a lot of IDs and need them to work quickly, you can send them to me and I will convert them to a list of slugs for you.

**Comment by endrjux** [comment]
Thanks, I've already manually converted them myself. It appears now #41 showed up for me as well.

**Comment by jakobkmar** [comment]
Yeah, unfortunately #41 is not a bug, just bad feedback which needs to be improved. A few days ago, Curseforge enabled the feature that projects cannot be downloaded by third party clients by default and only if the author explicitly enables it, the project can be downloaded.

You can either ask the author to enable that feature or try to find that mod on Modrinth.
